### ISSUE TITLE
Deprecation warning in Sketch 3.7

### ISSUE THREAD

**Issue by bomberstudios** [jamztang/MagicMirror#30]
Hi there,

I was doing my usual round of plugin testing, and MagicMirror seems to be working fine in 3.7. However, it's showing this deprecation warning, so you may want to take a look:

```
05/04/16 16:10:34,522 Sketch Beta[79203]: setPatternImage: is deprecated. Use setImage: instead with MSImageData «Scripting Legacy»
```

Hope it helps!

**Comment by jamztang** [comment]
Thanks @bomberstudios Will be looking at it!

**Comment by jamztang** [comment]
Fixed in v2.0.6! Cheers!
